tests/module/test_icp.py

```diff
@@ -0,0 +1,72 @@
+import torch
+import pypose as pp
+from torchvision.datasets.utils import download_and_extract_archive
+
+class TestICP:
+
+    def load_point_cloud(self):
+        download_and_extract_archive('https://github.com/Murphy41/laser-scan-pt/' \
+                                     'releases/download/v0.0/icp-test-data.pt.zip',\
+                                    './tests/module')
+        loaded_tensors = torch.load('./tests/module/icp-test-data.pt')
+        pc1 = loaded_tensors['pc1'].squeeze(-3)
+        pc2 = loaded_tensors['pc2'].squeeze(-3)
+        return pc1, pc2
+
+    def test_icp_laserscan(self):
```

Comment:
Formatting
See https://peps.python.org/pep-0008/#indentation

Reply:
Hi Zitong, is this comment for line 10 or other lines?